src/styles/main.scss

```diff
@@ -1,3 +1,117 @@
-body {
-  margin: 0;
+@import "variables";
+@import "stars";
+@import "header";
+@import "nav";
+@import "reset";
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  background: $background--color--card;
+  border: $border--thickness solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  width: 200px;
+  transition: transform $transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 32px auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color--text;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    display: flex;
+    font-size: 10px;
+    color: $color--text--title;
+    font-weight: 400;
+    margin-bottom: 16px;
+    align-self: center;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    display: flex;
+    font-size: 10px;
+    color: $color--text;
+    font-weight: 400;
+    margin-bottom: 26px;
```

Comment:
still not fixed
![Screenshot from 2023-06-13 15-37-48](https://github.com/mate-academy/layout_catalog/assets/33029864/61a28721-83eb-45e9-93f8-ba9a59665127)

```suggestion
    justify-content: space-between;
    margin-bottom: 26px;
  }

  &__reviews {
    display: flex;
    font-size: 10px;
    color: $color--text;
    font-weight: 400;
```